### DISCUSSION TITLE
magento-wysiwyg-fix

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#1962] [Extensions & Modules]
looking on the net for a solution to a different problem I came across this article

[articolo](https://www.ctidigital.com/blog/fixing-magento-improving-performance-of-the-wysiwyg-editor/)

I am not a developer and I do not know if this solution has already been used in OpenMage and I do not even know if it is needed, in any case if needed I considered it correct to share it.

[magento-wysiwyg-fix](https://github.com/ctidigital/magento-wysiwyg-fix)

**Comment by sreichel** [comment]
Should be fixed with #984
